### PR TITLE
Update docs on instance-level updates vs. data loss race condition

### PR DESCRIPTION
**Pull request by a1exsh**
Recommend to use the latest Taupage AMI and set erase_on_boot flag explicitly
to false to avoid data loss and to remove the need for creating and assigning
instance profile, since w/o the flag Taupage will try list the volume's Tags.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -179,10 +179,12 @@ to scale up EC2 instances or update Taupage AMI.
...
-#. Be sure to reuse the IP of the node you just terminated on the new node and to change the instance type (and/or pick a different Taupage AMI).
-#. While the new instance is spinning up, attach the (now detached) data volume to the new instance.  Use ``/dev/sdf`` as the device name. **Warning: it was observed that this might lead to failure to mount the filesystem on the volume and loss of all data on it.  Improved procedure needs to be tested.**
+#. **Use the latest available Taupage AMI version.  Older versions are subject to data loss race conditions when attaching EBS volumes.**
+#. Be sure to reuse the private IP of the node you just terminated on the new node.
+#. In the 'Instance Details' section, edit 'User Data' to add ``erase_on_boot: false`` flag under ``mounts: /var/lib/cassandra``.  See documentation of Taupage_ for detailed decsription and syntax example.  The docker image version being used can also be updated in this section, however, it is recommended to avoid changing multiple things at a time.  Also, docker image can be update without terminating the instance, by stopping and starting it with updated 'User Data' instead.
```
Comment (slitsche) [inline README.rst]: `decsription` -> description

**Comment by hjacobs** [comment]
Can we merge this? I can't rely verify if the instructions are correct or not.

**Comment by a1exsh** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Comment by MALPI** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Comment by MALPI** [comment]
👍

**Comment by slitsche** [comment]
👍

**Pull request merged**